openmmtools/integrators.py

```diff
@@ -585,14 +585,15 @@ class NoseHooverChainVelocityVerletIntegrator(ThermostatedIntegrator):
 
     Create a velocity Verlet integrator with Nosé-Hoover chain thermostat.
 
+    >>> system = testsystems.WaterBox()
```

Comment:
I think this should be called something like `waterbox` or `testsystem` and then the quantity passed into the c'tor below should be `waterbox.system`.  I could be wrong, but that's my memory of it.